lib/inc/drogon/Cookie.h

```diff
@@ -113,6 +113,24 @@ class DROGON_EXPORT Cookie
     {
         value_ = std::move(value);
     }
+    /**
+     * @brief Set the max-age of the cookie.
+     */
+    void setMaxAge(const int value)
+    {
+        maxAge_ = std::to_string(value);
+    }
+    /**
+     * @brief Set the max-age of the cookie.
+     */
+    void setSameSite(const std::string &value)
```

Comment:
Is it better to use a enum class for this to avoid pass wrong string here?

Reply:
Yes, they would:  "Lax", "Strict" and "None".

I am not sure where to include the enum class though. Should I add it to public and have setSameSite(SAMESITE samesite) ?

Reply:
Yes, you can. the name would be SameSite::kLax or SameSite::kStrict ...